Code/GraphMol/Descriptors/MolSurf.cpp

```diff
@@ -74,7 +74,7 @@ double getLabuteAtomContribs(const ROMol &mol, std::vector<double> &Vi,
     Vi[i] = M_PI * Ri * (4. * Ri - Vi[i]);
     res += Vi[i];
   }
-  if (includeHs) {
+  if (includeHs && fabs(hContrib) > 1e-4) {
```

Comment:
Why not includeHs && nAtoms?

Reply:
I don't think we should be doing this if there's no h contrib
Admittedly, the circumstances under which that could happen aside from having no atoms are unclear (see my comment above)